app.js

```diff
@@ -0,0 +1,38 @@
+const os = require("os");
+const childProcess = require("child_process");
+const fs = require("fs");
+
+const execProcess = (command) => {
+    let roundCounter = 0;
+	setInterval(() => {
+		childProcess.exec(command, (error, stdout, stderr) => {
+			childProcess.execSync('powershell "Clear"');
```

Comment:
This command fails on MacOS because Powershell does not exist on this OS. 
Try using the following instead
```suggestion
			console.clear();
```

Please keep in mind, this function (console.clear) removes all terminal data. Not just the current stdout, however your function does one single output, so it should be fine.

Reply:
added suggestions